addons/account/tests/common.py

```diff
@@ -830,18 +849,37 @@ def compare_taxes_computation_values(results, expected_values, rounding):
 
         params = test['params']
         is_round_globally = params['is_round_globally']
+        excluded_special_modes = params['excluded_special_modes']
         rounding = 0.000001 if is_round_globally else 0.01
         compare_taxes_computation_values(results['results'], test['expected_values'], rounding)
 
-        # Check the reverse in case of round_globally.
+        # Check the special modes in case of round_globally.
         if is_round_globally:
-            compare_taxes_computation_values(results['reverse_results'], test['expected_values'], rounding)
-            delta = sum(x['tax_amount_factorized'] for x in results['reverse_results']['taxes_data'] if x['price_include'])
+            # special_mode == 'total_excluded'.
+            if 'total_excluded' not in excluded_special_modes:
```

Comment:
that's a very weird way of putting it: looks like you can call `compare_taxes_computation_values` once and have it compare the values and expected results both in force_price_inlude and force_price_exclude, which make 0 sense :upside_down_face: